code/package.json

```diff
@@ -1,25 +1,23 @@
 {
   "name": "esp8266-filesystem-builder",
-  "version": "0.2.1",
+  "version": "0.2.2",
```

Comment:
Is version even required?

Reply:
npm docs says no. and neither is name. both can be safely removed.